src/gp_doctest.cpp

```diff
@@ -18,191 +18,78 @@ GPInstance MakeHelloGPInstance() {
   inst.ReadFastaFile("data/hello.fasta");
   inst.ReadNewickFile("data/hello_rooted.nwk");
   inst.MakeEngine();
-  EigenVectorXd branch_lengths(4);
+  EigenVectorXd branch_lengths(5);
   // Order set by HelloGPCSP.
-  branch_lengths << 0.113, 0.15, 0.1, 0.22;
+  branch_lengths << 0, 0.113, 0.22, 0.1, 0.15;
   inst.GetEngine()->SetBranchLengths(branch_lengths);
   return inst;
 }
 
-TEST_CASE("GPInstance: straightforward classical likelihood calculation") {
```

Comment:
It seems like we're losing a teaching tool by getting rid of this test. Would it be hard to update to bring it into your framework?